### ISSUE TITLE
Sprite Glitches

### ISSUE THREAD

**Issue by BlueViper85** [Dessyreqt/alttprandomizer#46]
I experienced two sprite glitches in C4496861 (v5).  One is fairly minor, the other could be problematic, but still fairly inconsequential since it it still shows the correct item once picked up.
1. Digging game showed three green bombs instead of the Power Gloves.
2. Minor, but on the bombos tablet the red rupee that fell was only showing half of the rupee as it fell.

**Comment by KatDevsGames** [comment]
<pre>3. I saw a blue potion bottle get cut clear in half when it was sitting on the pyramid.</pre>

CURRENTLY FIXED - LEAVE OPEN PENDING #1

**Comment by BlueViper85** [comment]
Watching through the "Let's Dev" VoD while you're working on the digging game with Titan's Mitts, 

I remembered that, I noticed with the digging game during other testing I noticed that it seemed like the location I dug an item from OR the direction the item moved matter.  The mushroom was in the digging game and the first time I dug it up I was in the middle of the digging game area and digging to the right and the mushroom appeared fine.  But when I loaded a save state later and did the game again I was digging near the entrance and facing towards the left when I dug it up the Mushroom had a different sprite.

I know this isn't specific, but I wanted to note it here so it's known and hopefully so I won't forget (or as a reminder if I do).  I'll test further when I'm able to tonight and update with specific details.
